go/preload/main/preload.go

```diff
@@ -171,7 +182,7 @@ func main() {
 	sctWriterWG.Add(1)
 	go sctWriterJob(addedCerts, sctWriter, &sctWriterWG)
 
-	submitLogClient := client.New(*targetLogUri)
+	submitLogClient := client.New(*targetLogUri, &http.Client{})
```

Comment:
There's another here, but I *think* you might be okay if you reused the `httpclient.Transport`?